src/olympia/reviewers/management/commands/review_reports_sql/addon/breakdown.sql

```diff
@@ -2,30 +2,46 @@ SELECT 'All Reviewers' AS `Group`,
        IFNULL(FORMAT(SUM(aa.weight), 0), '-') AS `Total Risk`,
        IFNULL(FORMAT(AVG(aa.weight), 2), '-') AS `Average Risk`,
        FORMAT(COUNT(*), 0) AS `Add-ons Reviewed`
-FROM `reviewer_scores` `rs`
-LEFT JOIN `editors_autoapprovalsummary` `aa` ON `aa`.`version_id` = `rs`.`version_id`
-JOIN `users` `u` ON `u`.`id` = `rs`.`user_id`
-WHERE DATE(`rs`.`created`) BETWEEN @WEEK_BEGIN AND @WEEK_END
-  AND `u`.`deleted` = 0
+FROM (
+  SELECT DISTINCT `created`, `user_id`, `action`, MAX(`id`) AS `id`
+  FROM `log_activity`
+  WHERE DATE(`created`) BETWEEN @WEEK_BEGIN AND @WEEK_END
+  /* The type of review, see constants/activity.py */
+  AND `action` IN (21, 43, 144, 165)
+  GROUP BY `created`, `user_id`, `action`
```

Comment:
`DISTINCT` *and* `GROUP BY` ? 

Also, we should add a comment here documenting that this only works because we explicitly set `created` to be the exact same date for actions that record multiple `ActivityLog`s.

Reply:
https://github.com/mozilla/addons-server/pull/20060/commits/853aa0071b02f8254638682381d2c7a1046f2d9d

Reply:
and https://github.com/mozilla/addons-server/pull/20060/commits/8618d2287f584529a2163cb7c6890175b690c2ee for the comments